xrootd.spec

```diff
@@ -39,7 +37,7 @@ cmake ../ \
   -DENABLE_CRYPTO=TRUE \
   -DCMAKE_SKIP_RPATH=TRUE \
   -DENABLE_PYTHON=TRUE \
-  -DCMAKE_PREFIX_PATH="${PYTHON_ROOT}"
```

Comment:
@iarspider , looks like we might need these flags https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_15_0_X/master/xrootd.spec#L47-L48